File: src/app/models/staticData.ts
import { Ingredients, IngredientTypes } from "./Ingredient";
import { Items } from "./Item";
import { DrinkTypes, ItemGroups, Sides } from "./ItemGroup";
import { Users } from "./User";
import { Contacts } from "./Contact";

export interface StaticData {
  items: Items,
  itemGroups: ItemGroups,
  ingredients: Ingredients,
  ingredientTypes: IngredientTypes,
  drinkTypes: DrinkTypes
}

export interface userData {
  users: Users
  contacts: Contacts
}

File: WebAPI/Models/StaticData.cs
using System.Text.Json;
using WebAPI.Data;

namespace WebAPI.Models
{
  public class StaticData
  {
    public StaticData() { }
    public StaticData(
      Item[] items, ItemGroup[] itemGroups, Ingredient[] ingredients,
      IngredientType[] ingredientTypes, DrinkType[] drinkTypes)
    {
      Items = items;
      ItemGroups = itemGroups;
      Ingredients = ingredients;
      IngredientType = ingredientTypes;
      DrinkTypes = drinkTypes;
    }

    public Item[] Items { get; set; }
    public ItemGroup[] ItemGroups { get; set; }
    public Ingredient[] Ingredients { get; set; }
    public IngredientType[] IngredientType { get; set; }
    public DrinkType[] DrinkTypes { get; set; }

    // send the error as a JON object
    public override string ToString()
    {
      // sets the properties in camel case ... allows use of
      // standard notation for both api & client-side consumption
      var options = new JsonSerializerOptions()
      {
        PropertyNamingPolicy = JsonNamingPolicy.CamelCase
      };
      return JsonSerializer.Serialize(this, options);
    }
  }
}
